### PR TITLE
Removed default values from flags

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

**Review by mingazhev — COMMENTED** [review]
You should up the package version. Please find how it is typically done in git. There should be at least:
- package version update
- release notes

File: HappyTravel.EdoContracts/Accommodations/Enums/BoardBasisTypes.cs [hunk 1]
```diff
@@ -7,9 +7,6 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum BoardBasisTypes
     {
-        [Description("Not specified")]
-        NotSpecified = 0,
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Enums/BoardBasisTypes.cs:11]: This must not be removed

File: HappyTravel.EdoContracts/Accommodations/Enums/LocationDescriptionCodes.cs [hunk 2]
```diff
@@ -7,8 +7,6 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum LocationDescriptionCodes
     {
-        Unspecified = 0,
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Enums/LocationDescriptionCodes.cs:10]: This must not be removed

File: HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs [hunk 3]
```diff
@@ -7,8 +7,6 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum RoomTypes
     {
-        [Description("Not specified")]
-        NotSpecified = 0,
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs:11]: This must not be removed

File: HappyTravel.EdoContracts/Accommodations/Enums/TextualDescriptionTypes.cs [hunk 4]
```diff
@@ -6,7 +6,6 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum TextualDescriptionTypes
     {
-        Unspecified = 0,
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Enums/TextualDescriptionTypes.cs:9]: This must not be removed

File: HappyTravel.EdoContracts/GeoData/Location.cs [hunk 5]
```diff
@@ -21,8 +21,8 @@ public Location(string name, string locality, string country, in GeoPoint coordi
         }
 
 
-        public Location(in GeoPoint coordinates, int distance, LocationTypes type = LocationTypes.Unknown)
-            : this(string.Empty, string.Empty, string.Empty, coordinates, distance, PredictionSources.NotSpecified, type)
+        public Location(in GeoPoint coordinates, int distance, LocationTypes type = default)
```
Comment (kirillta) [inline HappyTravel.EdoContracts/GeoData/Location.cs:24]: I think the default here is not an option after changes

**Review by kirillta — CHANGES_REQUESTED** [review]

File: HappyTravel.EdoContracts/Accommodations/Enums/BoardBasisTypes.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/BoardBasisTypes.cs:11]: Returned

File: HappyTravel.EdoContracts/Accommodations/Enums/LocationDescriptionCodes.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/LocationDescriptionCodes.cs:10]: Returned

File: HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs (hunk 3 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs:11]: Returned

File: HappyTravel.EdoContracts/Accommodations/Enums/TextualDescriptionTypes.cs (hunk 4 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/TextualDescriptionTypes.cs:9]: Returned

File: HappyTravel.EdoContracts/GeoData/Location.cs (hunk 5 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/GeoData/Location.cs]: fixed

**Comment by DFSko** [comment]
> You should up the package version. Please find how it is typically done in git. There should be at least:
> 
> * package version update
> * release notes

fixed

**Review by kirillta — APPROVED** [review]

File: HappyTravel.EdoContracts/Accommodations/Enums/PropertyTypes.cs [hunk 6]
```diff
@@ -8,8 +8,8 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     [Flags]
     public enum PropertyTypes
     {
-        Any = 0,
-        Hotels = 1,
-        Apartments = 2
+        Any = 1,
+        Hotels = 2,
+        Apartments = 3
     }
```
Comment (hasmikharutyunyan) [inline HappyTravel.EdoContracts/Accommodations/Enums/PropertyTypes.cs:14]: Flag can not have value 3

File: HappyTravel.EdoContracts/Accommodations/Enums/BookingStatusCodes.cs [hunk 7]
```diff
@@ -13,45 +13,45 @@ public enum BookingStatusCodes
         ///     that something went wrong. A manual intervention is required.
         /// </summary>
         [Description("Internal Processing")]
-        InternalProcessing = 0,
+        InternalProcessing = 1,
```
Comment (hasmikharutyunyan) [inline HappyTravel.EdoContracts/Accommodations/Enums/BookingStatusCodes.cs:16]: This enum is not flag

Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/BookingStatusCodes.cs:16]: > This enum is not flag

Remove "Any"?

File: HappyTravel.EdoContracts/Accommodations/Enums/PropertyTypes.cs (hunk 6 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Enums/PropertyTypes.cs:14]: > Flag can not have value 3

Remove "Any"?

File: HappyTravel.EdoContracts/General/Enums/FieldTypes.cs [hunk 8]
```diff
@@ -2,7 +2,7 @@
 {
     public enum FieldTypes
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/General/Enums/FieldTypes.cs:3]: Is this a flag?

File: HappyTravel.EdoContracts/General/Enums/PassengerTitles.cs [hunk 9]
```diff
@@ -7,9 +7,6 @@ namespace HappyTravel.EdoContracts.General.Enums
     [JsonConverter(typeof(StringEnumConverter))]
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/General/Enums/PassengerTitles.cs:7]: This is not a flag

File: HappyTravel.EdoContracts/General/Enums/PassengerTitles.cs [hunk 10]
```diff
@@ -7,9 +7,6 @@ namespace HappyTravel.EdoContracts.General.Enums
...
     public enum PassengerTitles
     {
-        [Description("")]
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/General/Enums/PassengerTitles.cs:10]: This is not a flag

File: HappyTravel.EdoContracts/General/Enums/PaymentMethods.cs [hunk 11]
```diff
@@ -8,24 +8,24 @@ namespace HappyTravel.EdoContracts.General.Enums
     public enum PaymentMethods
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/General/Enums/PaymentMethods.cs:8]: This is not a flag

File: HappyTravel.EdoContracts/GeoData/Enums/LocationTypes.cs [hunk 12]
```diff
@@ -9,8 +9,6 @@ namespace HappyTravel.EdoContracts.GeoData.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum LocationTypes
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/GeoData/Enums/LocationTypes.cs:10]: This is not a flag

File: HappyTravel.EdoContracts/GeoData/Enums/PredictionSources.cs [hunk 13]
```diff
@@ -7,8 +7,6 @@ namespace HappyTravel.EdoContracts.GeoData.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum PredictionSources
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/GeoData/Enums/PredictionSources.cs:8]: This is not a flag

File: HappyTravel.EdoContracts/General/Enums/FieldTypes.cs (hunk 8 above)
Comment (mingazhev) [reply HappyTravel.EdoContracts/General/Enums/FieldTypes.cs:3]: This is not a flag

File: HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs [hunk 14]
```diff
@@ -8,19 +8,19 @@ namespace HappyTravel.EdoContracts.Accommodations.Enums
     public enum RoomTypes
     {
         [Description("Not specified")]
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/Accommodations/Enums/RoomTypes.cs:10]: This is not a flag

**Review by mingazhev — CHANGES_REQUESTED** [review]
Changes should be only in flags

**Pull request closed without merging**